finality-aleph/src/lib.rs

```diff
@@ -189,15 +189,16 @@ impl From<SpawnTaskHandle> for SpawnHandle {
     }
 }
 
-impl aleph_bft::SpawnHandle for SpawnHandle {
```

Comment:
Why you changed this since in the implementation of `aleph_bft::SpawnHandle` trait you just call them.

Reply:
That block was unnecessary - .await moves its Future.

Reply:
I would say even heavily depending on AlephBFT here and in `party.rs` is fine, and then there would be no need for accessing internal fields, just use the trait methods. The advantage of less boilerplate seems worth the minimal cost to me.

Reply:
I can't fully agree but in this case it doesn't matter that much. Generally, it's easier to maintain a stable internal api that just needs to be adapted to an external trait, than directly using some external trait. Any change in that external trait requires changes everywhere it was used internally and you have no "control" over it since you do not own it. When you just adapt to it, only the adapting part needs to be changed.

Reply:
That is a very good point, I just feel in our case the integration with AlephBFT is tight enough that any changes to the API will require quite a lot of work anyway. And we will most likely keep similar style and approaches in both repositories, so we will likely want to change the interface to mimic the one in AlephBFT anyway.

Eh, actually I don't have very strong feelings one way or another in this case, so I would say it's up to @mike1729. You have my approve anyway.

Reply:
Now it uses alephbft's trait instead.